File: 70. Climbing Stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n == 1)
{
      return 1;
 }
        int a = 1;
        int b = 1;
        int sum = 0;
        for(int i=0;i<n-1;i++)
        {
            sum = a + b;
            a = b;
            b = sum;
        }
        
        return sum;
    }
};

File: 70. Climbing Stairs.java
class Solution {
    public int climbStairs(int n) {
        
        int a = 1;
        int b = 1;

        for (int i = 0; i < n; i++) {
            int temp = a;
            a = b;
            b = temp + b;
        }

        return a;
    
    }
}
